payment/src/main/java/com/kyriba/conference/payment/service/NoSuchPaymentException.java

```diff
@@ -0,0 +1,9 @@
+package com.kyriba.conference.payment.service;
+
+import java.util.NoSuchElementException;
```

Comment:
what a reason to extend this exception?

Reply:
* Thrown by various accessor methods to indicate that the element being requested
 * does not exist.

This is from javadoc

Reply:
This is bad practice to extend such common JDK exceptions. Being thrown from streams or some other places could cause side effects if framework code catch them and process.

Reply:
fixed